lib/Drush/Commands/core/SanitizeCommands.php

```diff
@@ -0,0 +1,274 @@
+            $this->sanitizeTableColumn($table,  $field_name . '_value', $randomizer->paragraphs(2));
+            $this->sanitizeTableColumn($table,  $field_name . '_summary', $randomizer->name(255));
+            break;
+        }
+      }
+    }
+  }
+
+  /**
+   * Replaces all values in given table column with the specified value.
+   *
+   * @param string $table
+   *   The database table name.
+   * @param string $column
+   *   The database column to be updated.
+   * @param $value
+   *   The new value.
+   */
+  public function sanitizeTableColumn($table, $column, $value) {
+    $table_name_wrapped = $this->wrapTableName($table);
+    $sql = "UPDATE $table_name_wrapped SET $column='$value';";
+    drush_sql_register_post_sync_op($table.$column, dt("Replaces all values in $table table with the same random long string."), $sql);
+  }
+
+  /**
+   * Truncates the session table.
+   */
+  public function sanitizeSessions() {
+    // Seems quite portable (SQLite?) - http://en.wikipedia.org/wiki/Truncate_(SQL)
+    $table_name = $this->wrapTableName('sessions');
+    $sql_sessions = "TRUNCATE TABLE $table_name;";
+    drush_sql_register_post_sync_op('sessions', dt('Truncate Drupal\'s sessions table'), $sql_sessions);
+  }
+
+  /**
+   * Sanitizes comments_field_data table.
+   */
+  public function sanitizeComments() {
+
+    $comments_enabled = $this->query("SHOW TABLES LIKE 'comment_field_data';");
+    if (!$comments_enabled) {
+      return;
+    }
+
+    $comments_table = $this->wrapTableName('comment_field_data');
+    $sql_comments = "UPDATE $comments_table SET name='Anonymous', mail='', homepage='http://example.com' WHERE uid = 0;";
+    drush_sql_register_post_sync_op('anon_comments', dt('Remove names and email addresses from anonymous user comments.'), $sql_comments);
+
+    $sql_comments = "UPDATE $comments_table SET name=CONCAT('User', `uid`), mail=CONCAT('user+', `uid`, '@example.com'), homepage='http://example.com' WHERE uid <> 0;";
+    drush_sql_register_post_sync_op('auth_comments', dt('Replace names and email addresses from authenticated user comments.'), $sql_comments);
+  }
+
+  /**
+   * Wraps a table name in brackets if a database prefix is being used.
+   *
+   * @param string $table_name
+   *   The name of the database table.
+   *
+   * @return string
+   *   The (possibly wrapped) table name.
+   */
+  public function wrapTableName($table_name) {
+    if ($this->wrap) {
+      $processed = '{' . $table_name . '}';
+    }
+    else {
+      $processed = $table_name;
+    }
+
+    return $processed;
+  }
+
+  /**
+   * Executes a sql command using drush sqlq and returns the output.
+   *
+   * @param string $query
+   *   The SQL query to execute. Must end in a semicolon!
+   *
+   * @return string
+   *   The output of the query.
+   */
+  protected function query($query) {
+    $current = drush_get_context('DRUSH_SIMULATE');
+    drush_set_context('DRUSH_SIMULATE', FALSE);
+    $success = $this->executeQuery($query);
```

Comment:
Do we not need to handle query failure here? Also, can we merge query() and doQuery() especially since SQL API already has a query(). We now have 2 similar methods. Just asking.